cpu/cpu_windows.go

```diff
@@ -19,16 +21,26 @@ var cpuMap = map[string]string{
 	"machdep.cpu.stepping":     "stepping",
 }
 
+// Values that need to be multiplied by the number of physical processors
+var perPhysicalProcValues = []string{
+	"cpu_cores",
+	"cpu_logical_processors",
+}
+
 func getCpuInfo() (cpuInfo map[string]string, err error) {
 
 	cpuInfo = make(map[string]string)
 
-	cpu, err := utils.WindowsWMICommand("CPU",
+	cpus, err := utils.WindowsWMIMultilineCommand("CPU",
 		"CurrentClockSpeed", "Name", "NumberOfCores",
 		"NumberOfLogicalProcessors", "Caption", "Manufacturer")
 	if err != nil {
 		return
 	}
+        // each line represents a different CPUx
```

Comment:
tiny formatting issue here (extra whitespace before `//`). Could you run the file through `go fmt` to make sure there's no other formatting issue?